packages/create-react-native-library/src/template.ts

```diff
@@ -137,10 +142,12 @@ export function generateTemplateConfiguration({
               .replace(/[^a-z0-9](\w)/g, (_, $1) => $1.toUpperCase())
               .slice(1)}`,
       package: pack,
+      package_array: JSON.stringify(pack.split('.')),
```

Comment:
Since this is a single-use variable and is derived from another variable, prefer writing the split logic in the usage site directly instead of new config